Homeworks/test_module14.py

```diff
@@ -0,0 +1,82 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import pytest
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+username = "notabot"
+password = "1234"
+
+@pytest.fixture
+def browser():
+    driver = webdriver.Chrome()
+    yield driver
+    driver.quit()
+
+
+def test_navigation(browser):
+    browser.maximize_window()
```

Comment:
it's better move this to setup fixture because it's not the test step but the part of environment preparation